marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/model/DeleteModelTest.java

```diff
@@ -0,0 +1,93 @@
+package com.marklogic.hub.central.controllers.model;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.FailedRequestException;
+import com.marklogic.client.document.GenericDocumentManager;
+import com.marklogic.hub.DatabaseKind;
+import com.marklogic.hub.central.controllers.ModelController;
+import com.marklogic.hub.impl.EntityManagerImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.test.context.support.WithMockUser;
+
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DeleteModelTest extends AbstractModelTest {
+
+    @Autowired
+    ModelController controller;
+
+    @Test
+    @WithMockUser(roles = {"writeEntityModel"})
```

Comment:
WithMockUser shouldn't be needed anymore, right?

Reply:
It's required to make spring security happy as this is not using the MockMvc test framework.